doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,2 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
```

Comment:
@rybakit We only changed the default directory for now. If you want to customize the namespace or another setting of the bundle I suggest you take a look at the options at http://symfony.com/doc/current/bundles/DoctrineMigrationsBundle/index.html#configuration

/cc @fabpot @stof @javiereguiluz Might be a idea to add `"env": {}` to the manifest file with a link to `doc/current/bundles/DoctrineMigrationsBundle/`?